blog/disable-alerts-health-management-api.md

```diff
@@ -0,0 +1,151 @@
+Specifically, the API allows you to:
+- Disable health checks completely. Alarm conditions will not be evaluated at all and no entries will be added to the alarm log.
+- Silence alarm notifications. Alarm conditions will be evaluated, the alarms will appear in the log and the Netdata UI will show the alarms as active, but no notifications will be sent.
+- Disable or Silence specific alarms that match selectors on alarm/template name, chart, context, host and family.
+
+The API is available by default, but it is protected by an api authorization token that is stored in the file you will see in the following entry of http://NODE:19999/netdata.conf:
+```yaml
+[registry]
+   # netdata management api key file = /var/lib/netdata/netdata.api.key
+```
+ 
+You can access the API via GET requests, by adding the bearer token to an Authorization http header, like this:
+```bash
+curl "http://NODE:19999/api/v1/manage/health?cmd=RESET" -H "X-Auth-Token: Mytoken"
+```
+By default access to the health management API is only allowed from localhost. Accessing the API from anything else will return a 403 error with the message You are not allowed to access this resource. You can change permissions by editing the allow management from the variable in netdata.conf within the [web server access lists](https://learn.netdata.cloud/docs/agent/web/server#access-lists) section. See web server access lists for more information.
+
+The command RESET just returns Netdata to the default operation, with all health checks and notifications enabled. If you've configured and entered your token correctly, you should see the plain text response All health checks and notifications are enabled.
+
+**Note**: The Health Management API currently needs to be executed on every host / agent which is under maintenance and at some point in the future we will have a way of triggering maintenance modes from the **Netdata Cloud UI**.
+
+
+## Disable or silence all alarms
+
+If all you need is temporarily disable all health checks, then you issue the following before your maintenance period starts:
+```bash
+curl "http://NODE:19999/api/v1/manage/health?cmd=DISABLE ALL" -H "X-Auth-Token: Mytoken"
+```
+ 
+The effect of disabling health checks is that the alarm criteria are not evaluated at all and nothing is written in the alarm log. If you want the health checks to be running but to not receive any notifications during your maintenance period, you can instead use this:
+```bash
+curl "http://NODE:19999/api/v1/manage/health?cmd=SILENCE ALL" -H "X-Auth-Token: Mytoken"
+```
+Alarms may then still be raised and logged in Netdata, so you'll be able to see them via the UI.
```

Comment:
Should we also talk about how the user can just turn off notifications from the UI? They can do this when a maintenance window starts and turn them back on after its done. 

Would be good to drop a screenshot of this as well - the new one that Alexandros demo'd today.

Reply:
Yes, I think I will add this. Although, the users will currently need to do both - from the Agent and from the Cloud.

Reply:
Done, but with the old snapshots still.. I will take a look at what Alexandros delivers (if not already in).